### DISCUSSION TITLE
OpenCV

### DISCUSSION THREAD

**Discussion by Sinchana22BS** [Sinchana22BS/OpenCV#2] [Q&A] [answered]
why should we chose OpenCV for face detection?

**Comment by ravindrahvasudeva** [comment, marked as answer]
OpenCV is the huge open-source library for computer vision, machine learning, and image processing and is better for face detection compared to TensorFlow .

**Comment by Sinchana22BS** [comment]
what is the use of Haar cascade?

**Comment by ravindrahvasudeva** [comment]
Haar cascade is an algorithm that can detect objects in images, irrespective of their scale in image and location.
